test/JDBC/input/test_dynamic_local_vars.sql

```diff
@@ -0,0 +1,714 @@
+-- simple vars
```

Comment:
In the latest I see we are not getting $1 for explain output for all const nodes based on your changes. Since we will still get $ for targetlist vars, it would be good to have an explain test case side by side for both

Reply:
Added